contracts/utils/Strings.sol

```diff
@@ -77,23 +77,23 @@ library Strings {
 
             // compute log256(value), and add it to length
             uint256 valueCopy = value;
-            if (valueCopy >> 128 > 0) {
+            if (valueCopy > 1 << 128 ) {
                 valueCopy >>= 128;
                 length += 16;
             }
-            if (valueCopy >> 64 > 0) {
+            if (valueCopy > 1 << 64 ) {
                 valueCopy >>= 64;
                 length += 8;
             }
-            if (valueCopy >> 32 > 0) {
+            if (valueCopy > 1 << 32 ) {
                 valueCopy >>= 32;
                 length += 4;
             }
-            if (valueCopy >> 16 > 0) {
+            if (valueCopy > 1 << 16 ) {
                 valueCopy >>= 16;
                 length += 2;
             }
-            if (valueCopy >> 8 > 0) {
+            if (valueCopy > 1 << 8 ) {
```

Comment:
I'm quite certain that it should be `>=` everywhere, let's look at the example of `0x100`, it SHOULD trigger that `if`, but it won't, because it's `== 1 << 8`